app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+# Create like model
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] As I mentioned in the previous comment, the author does not exist in your database schema. Please correct the association between the Like and User models.